packages/palette-docs/src/layouts/MainLayout.tsx

```diff
@@ -79,59 +80,62 @@ const Layout = (props) => {
   const showTitle = type !== "page"
 
   return (
-    <Box ref={contentRef as any}>
-      <MetaTags title={name} />
-
-      <Box maxWidth={theme.breakpoints.md} margin="0 auto" px={[2, 4]}>
-        <GridColumns>
-          <Column
-            span={3}
-            position="sticky"
-            top={0}
-            height="100vh"
-            overflowY="auto"
-            borderRight="1px solid"
-            borderColor="black10"
-            pr={2}
-            py={4}
-            display={["none", "block"]}
-          >
-            <Sidebar />
-          </Column>
-
-          <Column span={7} py={4}>
-            <Box className="DocSearch-content">
-              {showTitle && (
-                <TitleArea
-                  name={name}
-                  source={source}
-                  status={status}
-                  editUrl={editUrl}
-                  mb={4}
-                />
-              )}
-
-              <MDXRenderer>{body}</MDXRenderer>
-            </Box>
-          </Column>
-
-          <Column
-            span={2}
-            position="sticky"
-            top={0}
-            height="100vh"
-            overflowY="auto"
-            py={4}
-            display={["none", "block"]}
-            borderLeft="1px solid"
-            borderColor="black10"
-            pl={2}
-          >
-            <TableOfContents headings={headings} />
-          </Column>
-        </GridColumns>
+    <>
+      <GlobalStyles />
```

Comment:
hat-tip to: https://www.ronnieho.dev/blog/styled-components/ssr-styled-components-gatsby/